log.graphqls

```diff
@@ -75,8 +75,23 @@ enum ContentType {
     YAML
 }
 
+input LogTestRequest {
+    # The log data of protocol https://github.com/apache/skywalking-data-collect-protocol/blob/e626ee04850703c220f64b642d2893fa65572943/logging/Logging.proto#
+    # in JSON format
+    log: String!
+    dsl: String!
+}
+
+type LogTestResponse {
+    log: Log # The final log if it will be persisted, this can be empty if the log is dropped.
+    metrics: [String!]
```

Comment:
Let's be clear what metrics string would be like. I think they are as same as MAL doc examples, right?